tsconfig.base.json

```diff
@@ -20,6 +20,7 @@
     "strict": true,
     "strictNullChecks": true,
     "alwaysStrict": true,
+    "strictPropertyInitialization": false,
```

Comment:
Why do we want this? https://www.typescriptlang.org/tsconfig/#strictPropertyInitialization

Reply:
<img width="888" alt="image" src="https://github.com/user-attachments/assets/665368cb-73d0-4b89-b9b7-77566ee8d53e">

There is a TS error. But I realized that it works in `twap` app, so I just copy-pasted tsconfig from here.